.github/workflows/node.yaml

```diff
@@ -299,6 +300,9 @@ jobs:
           - node-version: 14.x
             package-name: corelib
             send-coverage: true
+          - node-version: 14.x
+            package-name: shared-lib
```

Comment:
This should probably be in the `package-name` array above, as we should test it for the same runtimes as `server-core-integration`